hod/local.py

```diff
@@ -84,38 +87,109 @@ def known_cluster_labels():
         return []
 
 
-def cluster_env_file(label):
+def _cluster_info(label, info_file, contents=False):
     """
-    Return path to env file for cluster with specified label.
+    Return specified cluster info for cluster with specified label.
+    @param label: cluster label
+    @param info_file: type of info to return (env, jobid, ...)
+    @param contents: returns contents of info file rather than path to info file
     """
+    res = None
     labels = known_cluster_labels()
     if label in labels:
-        env_script = os.path.join(cluster_config_dir(), label, 'env')
-        if os.path.exists(env_script):
-            return env_script
+        info_file = os.path.join(cluster_info_dir, label, info_file)
+        if os.path.exists(info_file):
+            if contents:
```

Comment:
I'm not a fan. Just call this function and then open it and read it from outside.